console/src/components/IconButton/IconButton.less

```diff
@@ -0,0 +1,13 @@
+.button {
+
+	&.smallest {
+		height: 45px;
+		min-width: 0;
+		width: 45px;
+
+		.icon {
+			font-size: 39px;
+			height: 45px;
```

Comment:
The `icon` rules appear to cause unexpected alignment when using newer agate glyphs `<IconButton smallest icon="user" />`. This seems to be because this `smallest` icon dom size here is actually bigger than the `small` size, which is `36px`. I think we need to adjust this rule and add `width` and `line-height` using a value of half the `font-size` value (I believe that's the value used for general sizes, but I'd double-check that).